src/features/loading/LoadingScreen.tsx

```diff
@@ -267,6 +268,10 @@ class LoadingScreen extends Component<TLoadingProps, TLoadingState> {
     this.loadPastelConf(false)
   }
 
+  sleep = (ms: number) => {
+    return new Promise(resolve => setTimeout(resolve, ms))
+  }
```

Comment:
Seems like this function only used once. Is there a real need for it?

Reply:
it is only function for delaying a little time when sqlite file reading and writing is not done.